changelog.d/6749.feature

```diff
@@ -0,0 +1 @@
+Adds space list bottom sheet for new app layout
```

Comment:
I don't think we need to provide changelogs for features, which are not accessible for users 🤔